server/sonar-server/src/main/java/org/sonar/server/issue/IssueService.java

```diff
@@ -260,6 +257,7 @@ public DefaultIssue createManualIssue(String componentKey, RuleKey ruleKey, @Nul
 
       DefaultIssue issue = new DefaultIssueBuilder()
         .componentKey(component.getKey())
+        // TODO use rule type: type(rule.type())
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1135)